### DISCUSSION TITLE
were do i download it

### DISCUSSION THREAD

**Discussion by rhysmoon** [CodenameCrew/CodenameEngine#213] [General]
idk were to find it please help

**Comment by charlesisfeline** [comment]
[There's this thing called action builds.](https://github.com/FNF-CNE-Devs/CodenameEngine/actions)

**Comment by NexIsDumb** [comment]
use codename's website: https://codename-engine.com/

**Reply by THEkatinamicrowave** [reply]
I agree with this ^ ^ ^ ^ ^ ^^ ^^
